sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -761,7 +775,22 @@ synchronized void checkPendingLogs(@NonNull GroupState groupState) {
             triggerIngestion(groupState);
         } else if (pendingLogCount > 0 && !groupState.mScheduled) {
             groupState.mScheduled = true;
-            mAppCenterHandler.postDelayed(groupState.mRunnable, groupState.mBatchTimeInterval);
+            long delay = groupState.mBatchTimeInterval;
+            if (groupState.mBatchTimeInterval > MINIMUM_TRANSMISSION_INTERVAL) {
+                long now = System.currentTimeMillis();
+                long startTimer = SharedPreferencesManager.getLong(START_TIMER_PREFIX + groupState.mName);
```

Comment:
We must read it only once per process (after restart), the second time the interval is starting from current time, so not using the stored state.

Reply:
nvm, thats actually better to do it like that for large values in case SDK is idle.